### ISSUE TITLE
For some odd reason

### ISSUE THREAD

**Issue by hiimjustin000** [Cvolton/GMDprivateServer#608]
Android players are complaining that they lost their data when exiting out. I used IAD's All Admin GDPS for the GDPS. Is there anything to fix this?

**Comment by RedLime** [comment]
Did you change the package name?

**Comment by Lezzica** [comment]
If you changed the package name, then either you messed it up or forgot to rename all package name instances in the apk. Can be easily fixed using Notepad++ and it's "Replace in Files" feature.

**Comment by hiimjustin000** [comment]
Ahh ok but it says it stopped if I do so idk

**Comment by EMREOYUN** [comment]
It is known bug and there is no fix i think. My all servers apps are doing this.

**Comment by Cvolton** [comment]
> If you changed the package name, then either you messed it up or forgot to rename all package name instances in the apk. Can be easily fixed using Notepad++ and it's "Replace in Files" feature.
> If not, then I am not 100% sure what else the issue could be.

renaming all instances of the package name in the apk is not recommended, as it will lead to smali inconsistencies if done incorrectly, just make sure to replace it in the lib files

**Comment by EMREOYUN** [comment]
> > If you changed the package name, then either you messed it up or forgot to rename all package name instances in the apk. Can be easily fixed using Notepad++ and it's "Replace in Files" feature.
> > If not, then I am not 100% sure what else the issue could be.
> 
> renaming all instances of the package name in the apk is not recommended, as it will lead to smali inconsistencies if done incorrectly, just make sure to replace it in the lib files

Is there is any way to save the game(i mean not deleting save after leaving app) after package name changes?

**Comment by Cvolton** [comment]
> > > If you changed the package name, then either you messed it up or forgot to rename all package name instances in the apk. Can be easily fixed using Notepad++ and it's "Replace in Files" feature.
> > > If not, then I am not 100% sure what else the issue could be.
> > 
> > 
> > renaming all instances of the package name in the apk is not recommended, as it will lead to smali inconsistencies if done incorrectly, just make sure to replace it in the lib files
> 
> Is there is any way to save the game(i mean not deleting save after leaving app) after package name changes?

yes, just edit the package name in the libs appropriately, don't edit any smali files though

**Comment by SadShikawa** [comment]
Is it okay that I will "replace in files" the www.boomlings.com to my domain?

**Comment by fancythedeveloper** [comment]
Yes.
